### ISSUE TITLE
The validator requires the pair tags ever for SVG

### ISSUE THREAD

**Issue by TokugawaTakeshi** [validator/validator#1429]
I got the warning 

> Warning: Self-closing tag syntax in text/html documents is [widely discouraged](https://google.github.io/styleguide/htmlcssguide.html#Document_Type); it’s unnecessary and [interacts badly with other HTML features](https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=10809) (e.g., unquoted attribute values). 
> If you’re using a tool that injects self-closing tag syntax into all void elements, [without any option to prevent it from doing so](https://github.com/prettier/prettier/issues/5246), then consider switching to a different tool.

for the code:

```html
<!DOCTYPE html>
<html lang="en"><head>
  <title>Inbox items</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
</head>

<body>
  <svg class="NavigationMenu-Link-Icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 19.05">
     <path d="M19.7 7.05h-7.2L10.28.2a.3.3 0 0 0-.56 0L7.5 7.05H.3c-.29 0-.41.37-.17.53l5.82 4.23-2.22 6.85c-.09.27.22.5.45.33L10 14.76l5.82 4.23c.23.17.54-.06.45-.33l-2.22-6.85 5.82-4.23c.23-.17.11-.53-.17-.53Z"/>
  </svg>
  <span class="NavigationMenu-Link-Label">Inbox</span>         
</body>
</html>
```

To be honest I have not read the specification and [different specialists says different things](https://www.quora.com/Do-you-need-closing-tags-with-SVGs). So, If from the viewpoint of HTML validity self-closing SVG tags are valid, please make the validator ignore self-closing tags. Otherwise I'll go to report to [Pug](https://pugjs.org/api/getting-started.html) developers to about changed standards ...

> then consider switching to a different tool.

I don't want to disable the HTML validation just because of above issue ...

**Comment by sideshowbarker** [comment]
Thanks for reporting this. I agree this is a bug and I’m working on a fix for it right now. I’ll post another comment here as soon as I do actually have it fixed.

**Comment by TokugawaTakeshi** [comment]
@sideshowbarker 

Thank you too, for the processing of my report.

**Comment by sideshowbarker** [comment]
This should be fixed now — I just now deployed a fix, so the checker should no longer be warning about SVG or MathML elements that use self-closing-tag syntax.

Thanks again for catching this. If you find that the checker is still doing anything unexpected, please let me know.

**Comment by sahir72** [comment]
@sideshowbarker I got a new HTML Warnings against: 

<img /> , <br /> , <meta />
please share your releases document link about this...
